sigmastate/src/main/scala/sigmastate/lang/syntax/Exprs.scala

```diff
@@ -1,23 +1,23 @@
 package sigmastate.lang.syntax
 
-import fastparse.noApi._
+import fastparse._
+import ScalaWhitespace._
 import sigmastate._
 import sigmastate.Values._
-import sigmastate.lang.Terms.{ValueOps, Ident, Val}
+import sigmastate.lang.Terms.{Val, Ident, ValueOps}
 import sigmastate.lang._
 import sigmastate.lang.SigmaPredef._
 import sigmastate.lang.syntax.Basic._
 
 import scala.annotation.tailrec
-import scala.collection.mutable
+import scala.collection.compat.immutable.ArraySeq
 
 //noinspection ForwardReference,TypeAnnotation
 trait Exprs extends Core with Types {
```

Comment:
ScalaDoc missed

Reply:
fixed in https://github.com/ScorexFoundation/sigmastate-interpreter/commit/2cee01a333ebfa83b24d849b223d5eb89db8bb52